app/src/main/java/io/github/droidkaigi/confsched2017/repository/sessions/SessionsLocalDataSource.java

```diff
@@ -45,9 +45,7 @@ private Topic_Relation topicRelation() {
 
     @Override
     public Single<List<Session>> findAll(String languageId) {
-        return sessionRelation().selector().executeAsObservable().toList()
-                .subscribeOn(Schedulers.io())
-                .observeOn(AndroidSchedulers.mainThread());
```

Comment:
IMO: I think that the thread control should be decided by actual use place such as Activity, Fragment, Presenter etc.  Also, can't test Observable containing thread operation with Observable.test().